src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import './scss/App.scss';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] It would be better if your `App` component is class-based as one of the requirements. Kindly fix this minor issue. 😄 👍

Reply:
For this one, I was getting a linter error as it is a stateless component.
@BrianSammit

Reply:
Here is the rule it is bringing up `react/prefer-stateless-function`